### ISSUE TITLE
Maximum recursion depth exceeded during google/bigtable/v2 generation

### ISSUE THREAD

**Issue by busunkim96** [googleapis/gapic-generator-python#585]
Tested with 0.32.1 and 0.32.3

**Steps to reproduce**:

Edit [google/bigtable/v2/BUILD.bazel](https://github.com/googleapis/googleapis/blob/master/google/bigtable/v2/BUILD.bazel) with these Python rules
```starlark
##############################################################################
# Python
##############################################################################
# This library is using Python microgenerator.
# DO NOT OVERRIDE this Python section with autogenerated rules.
load(
    "@com_google_googleapis_imports//:imports.bzl",
    py_gapic_assembly_pkg = "py_gapic_assembly_pkg2",
    py_gapic_library = "py_gapic_library2",
)

py_gapic_library(
    name = "bigtable_py_gapic",
    srcs = [":bigtable_proto"],
)

# Open Source Packages
py_gapic_assembly_pkg(
    name = "bigtable-v2-py",
    deps = [
        ":bigtable_py_gapic",
    ],
)

```

Generate via bazel:

```
bazel build //google/bigtable/v2:bigtable-v2-py
```

Generation fails with the following stack trace (truncated for readability):
```
  File "/usr/local/google/home/busunkim/.cache/bazel/_bazel_busunkim/5f7ad274936c83fa5125630e373edd54/sandbox/linux-sandbox/2139/execroot/com_google_googleapis/bazel-out/host/bin/external/gapic_generator_python/gapic_plugin.runfiles/gapic_generator_python/gapic/utils/cache.py", line 39, in inner
    self._cached_values[fx.__name__] = fx(self)
  File "/usr/local/google/home/busunkim/.cache/bazel/_bazel_busunkim/5f7ad274936c83fa5125630e373edd54/sandbox/linux-sandbox/2139/execroot/com_google_googleapis/bazel-out/host/bin/external/gapic_generator_python/gapic_plugin.runfiles/gapic_generator_python/gapic/schema/wrappers.py", line 118, in mock_value
    answer = f'{self.type.ident}({sub.name}={sub.mock_value})'
  File "/usr/local/google/home/busunkim/.cache/bazel/_bazel_busunkim/5f7ad274936c83fa5125630e373edd54/sandbox/linux-sandbox/2139/execroot/com_google_googleapis/bazel-out/host/bin/external/gapic_generator_python/gapic_plugin.runfiles/gapic_generator_python/gapic/utils/cache.py", line 39, in inner
    self._cached_values[fx.__name__] = fx(self)
  File "/usr/local/google/home/busunkim/.cache/bazel/_bazel_busunkim/5f7ad274936c83fa5125630e373edd54/sandbox/linux-sandbox/2139/execroot/com_google_googleapis/bazel-out/host/bin/external/gapic_generator_python/gapic_plugin.runfiles/gapic_generator_python/gapic/schema/wrappers.py", line 118, in mock_value
    answer = f'{self.type.ident}({sub.name}={sub.mock_value})'
  File "/usr/local/google/home/busunkim/.cache/bazel/_bazel_busunkim/5f7ad274936c83fa5125630e373edd54/sandbox/linux-sandbox/2139/execroot/com_google_googleapis/bazel-out/host/bin/external/gapic_generator_python/gapic_plugin.runfiles/gapic_generator_python/gapic/utils/cache.py", line 39, in inner
    self._cached_values[fx.__name__] = fx(self)
  File "/usr/local/google/home/busunkim/.cache/bazel/_bazel_busunkim/5f7ad274936c83fa5125630e373edd54/sandbox/linux-sandbox/2139/execroot/com_google_googleapis/bazel-out/host/bin/external/gapic_generator_python/gapic_plugin.runfiles/gapic_generator_python/gapic/schema/wrappers.py", line 118, in mock_value
    answer = f'{self.type.ident}({sub.name}={sub.mock_value})'
  File "/usr/local/google/home/busunkim/.cache/bazel/_bazel_busunkim/5f7ad274936c83fa5125630e373edd54/sandbox/linux-sandbox/2139/execroot/com_google_googleapis/bazel-out/host/bin/external/gapic_generator_python/gapic_plugin.runfiles/gapic_generator_python/gapic/schema/metadata.py", line 79, in __str__
    if self.module_alias:
RecursionError: maximum recursion depth exceeded
--python_gapic_out: protoc-gen-python_gapic: Plugin failed with status code 1.
Target //google/bigtable/v2:bigtable-v2-py failed to build
```

CC @kolea2

**Comment by software-dov** [comment]
The rendering is getting caught on data.proto:RowFilter. This is a recursive message type via  the 'filter' oneof with recurrant case in 'chain', 'interleave', and 'condition' variants and nested types.

The solution is to have a hard recursion cap in the rendering itself. Fix is incoming.

**Comment by software-dov** [comment]
Fixed by #587
